pkgs/development/libraries/science/math/magma/default.nix

```diff
@@ -72,5 +72,8 @@ in stdenv.mkDerivation {
     maintainers = with maintainers; [ tbenst ];
   };
 
-  passthru.cudatoolkit = cudatoolkit;
+  passthru = {
+    # TODO: leave just cudaPackages
```

Comment:
What's the holdup on this change? Is this something you intend to do before merging or is this for the future?

Reply:
- Wasn't sure if someone relies on this passthru
- I was actually considering whether we should passthru our symlinkJoin under this name? I thought about reusing it for torchvision and torchaudio. Again, we can change this later